cpp/player.h

```diff
@@ -14,8 +14,11 @@ class Unit;
 
 class Player {
 public:
-	Player(unsigned int number);
+	Player(unsigned int number, std::string name);
 
+	/**
+	 * numbers 0-7
+	 */
 	const unsigned int player_number;
```

Comment:
nah, not just 0-7, it's uint128_t of course :grin:
but srsly, we should remove the 8 player restriction as soon as possible.